packages/snap-store-mobx/src/Search/Stores/SearchResultStore.ts

```diff
@@ -228,7 +263,7 @@ export class Variants {
 		}
 	}
 
-	public update(fromSelection: VariantSelection) {
+	public refineOptions(fromSelection: VariantSelection) {
```

Comment:
`refineSelections` and rename same named function below (inside VariantSelection) to `refineValues`.